web_export_view/static/src/js/web_export_view.js

```diff
@@ -65,7 +65,11 @@ odoo.define('web_export_view', function (require) {
                         }
                         else {
                             var text = $cell.text().trim();
-                            if ($cell.hasClass("o_list_number")) {
+                            // Search : for hour value
+                            if (text.search(":")) { 
```

Comment:
As far as I remember you still get the class for a number but you get `:` into it, right?
If so, this should be an exception for the other IF block (w/ proper comment):
```js
if ($cell.hasClass("o_list_number") && !text.search(":")):
    // here we have a number w/out time formatting (eg: 3:00)
```

Reply:
Maybe test `parseFloat()` results for NaN and in that case write text as-is? I don't think there's any benefit of testing for `:` presence only

Reply:
```js
parseFloat('03:00')
3
```
I  wish you were right :smile:

Reply:
🙈 javascript... maybe `parseFloat('03:00').toString() === '03:00'`? Yet this may not be super ok

Reply:
I meant 
```
if (parseFloat(text).toString() !== text) {
  // totally not a float
}
```

Reply:
Perhaps can use ```_.isNaN(+text)``` (ensure that text have `.` as decimal separator)

Reply:
> As far as I remember you still get the class for a number but you get `:` into it, right?
> If so, this should be an exception for the other IF block (w/ proper comment):
> 
> ```js
> if ($cell.hasClass("o_list_number") && !text.search(":")):
>     // here we have a number w/out time formatting (eg: 3:00)
> ```

it works for me